server/src/dp_queries/dp_querier.py

```diff
@@ -1,8 +1,10 @@
 from abc import ABC, abstractmethod
-from typing import List
+from typing import List, TypeAlias
 
 from private_dataset.private_dataset import PrivateDataset
 
+# https://stackoverflow.com/questions/51291722/define-a-jsonable-type-using-mypy-pep-526
+JSON: TypeAlias = dict[str, "JSON"] | list["JSON"] | str | int | float | bool | None 
```

Comment:
found this solution on stackoverflow for JSON type variable. Not totally sure if its the best solution

Reply:
I think it is better than what is there currently